config/config.py

```diff
@@ -38,8 +38,8 @@ class Development(Base):
 
     DEBUG = True
     # SQLALCHEMY_DATABASE_URI = "postgresql://postgres:postgres@database/cranecloud"
-    SQLALCHEMY_DATABASE_URI = os.getenv("DATABASE_URI", "postgresql:///cranecloud")
-    MONGO_URI = os.getenv("MONGO_URI", "mongodb://localhost:27017/cranecloud")
+    SQLALCHEMY_DATABASE_URI = os.getenv("DATABASE_URI", "postgresql:///cranecloud2")
```

Comment:
maintain `cranecloud` as the the db name and `cranecloud2` can be used in your local env